query-connector/docker-compose-dev.yaml

```diff
@@ -49,6 +49,8 @@ services:
       - hapi-fhir-server
 
   # Keycloak for authentication
+  # Note: you must have a local .env file with a parameter AUTH_DISABLED.
```

Comment:
Should this be added to the .env.sample?

Reply:
Good catch, will add that!